### ISSUE TITLE
html Lang attribute not seen correctly

### ISSUE THREAD

**Issue by brianread108** [validator/validator#2068]
### Summary

I get this:
```
Error: Bad value ARRAY(0x55d4b8bc3e98) for attribute lang on element [html](https://html.spec.whatwg.org/multipage/#the-html-element): Subtags must not exceed 8 characters in length.

From line 1, column 16; to line 2, column 36

TYPE html>↩<html lang="ARRAY(0x55d4b8bc3e98)" >↩<head
```
The browser console and the source code shows:
```
<html lang="en-GB"><head>
```
This is a perl mojolicious application and the actual mojo template code is this:
```
<html lang="<%= stash('locale') || 'en' %>" >
```
I am running vnu locally in a docker container inside an LXC. Normally this app is not available outside the local LAN.

I am passing the (local) URL to vnu for validation, and I can't see how or why the VNU would be seeing the internal mojolicious values rather than the result.

### URL for document that reproduces the problem

n/a (although it could be provided pro temps for a specific test.

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
n/a
```

### Steps to reproduce

n/a

### Expected behavior

n/a

### Actual behavior

n/a

**Comment by brianread108** [comment]
ok, found this:

```
root@librechat:~# curl -k https://sme11.thereadclan.me.uk/smanager/initial | head
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
<!DOCTYPE html>    0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
1<html lang="ARRAY(0x55d6bed75f10)" >
0<head>
0<title>Sme server 2 - Initial</title>
     <link rel="made" href="mailto:bugs%40koozali.org">
     <meta name="copyright" content="(head.tmpl)Copyright 2003-2004 Mitel Corporation">
8    
0    <link rel="icon" href="images/favicon.ico" type="image/x-icon">

```
I guess that means that the problem is my end not yours - sorry for the noise!!
